frontend/javascripts/oxalis/view/right-menu/meshes_view.js

```diff
@@ -131,6 +139,67 @@ class MeshesView extends React.Component<Props, { currentlyEditedMesh: ?MeshMeta
   };
 
   render() {
+    const moveToIsosurface = (seedPosition: Vector3) => {
+      Store.dispatch(setPositionAction(seedPosition));
+    };
+    const downloadButton = (segmentId: number) => (
```

Comment:
The naming of `downloadButton`, `refreshButton` and `deleteButton` is a bit unusual for these functions. Either rename them to `getDownloadButton`, `getRefreshButton` etc. or capitalize them to `DownloadButton` etc. In the latter case, one would use the functions as react components (i.e., `<DownloadButton segmentId=... />` on the caller side and `({segmentId}) => ...` on the definition side). The first variant is probably easier to adapt to :)